### PR TITLE
upgrade to new Uploadcare Api

### PR DESCRIPTION
**Pull request by dimaninc**

### CODE REVIEW

File: uploadcare.field.inc [hunk 1]
```diff
@@ -259,8 +264,8 @@ function uploadcare_field_widget_form(&$form, &$form_state, $field, $instance, $
 function uploadcare_field_widget_form_prefix() {
   $public_key = variable_get('uploadcare_public_key');
   $secret_key = variable_get('uploadcare_secret_key');
-  $api = new Uploadcare_Api($public_key, $secret_key);
-  return $api->widget->getScriptTag('0.8.1.2');
+  $api = new Uploadcare\Api($public_key, $secret_key);
+  return $api->widget->getScriptTag('2.5.1');
```
Comment (dmitry-mukhin) [inline uploadcare.field.inc]: 2.5.5 is the latest one

Comment (dmitry-mukhin) [inline uploadcare.field.inc]: and please remove magic numbers and add something more manageable

**Comment by dimaninc** [comment]
done

File: uploadcare.field.inc [hunk 2]
```diff
@@ -259,8 +264,8 @@ function uploadcare_field_widget_form(&$form, &$form_state, $field, $instance, $
 function uploadcare_field_widget_form_prefix() {
   $public_key = variable_get('uploadcare_public_key');
   $secret_key = variable_get('uploadcare_secret_key');
-  $api = new Uploadcare_Api($public_key, $secret_key);
-  return $api->widget->getScriptTag('0.8.1.2');
+  $api = new Uploadcare\Api($public_key, $secret_key);
+  return $api->widget->getScriptTag(Uploadcare\Widget::version);
```
Comment (dmitry-mukhin) [inline uploadcare.field.inc]: `Uploadcare\Widget::version` seems to be redundant

**Comment by dmitry-mukhin** [comment]
otherwise, ok

**Pull request merged**